Create analytics.md (#21)
ScootKit/scnx-docs: i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds /analytics.md

---
sidebar_position: 2
---

# Serverstatistiken

Schalte detaillierte Statistiken über deinen Server frei – erhalte Infos, welche Mitglieder, Kanäle und an welchen Tagen auf deinem Server Aktivitäten sind.

> Schau dir die Mitgliederaktivität an und erhalte eine Prognosen für die Zukunft 🔮

Reviewer: ```suggestion
Schau dir die Mitgliederaktivität an und erhalte Prognosen für die Zukunft 🔮
```

<IncludedInPlan data={{PRO: true, ACTIVE_GUILD: true, STARTER: true}}/>

:::tip Datenschutz liegt uns am Herzen
Du fragst dich, welche Daten wir über deinen Server speichern und wie wir sie verwenden? Bitte lese 
unseren [FAQ-Artikel](https://faq.scnx.app/datenerhebung-bei-statistiken/), um mehr zu erfahren.

**tl;dr**: Wir speichern Daten über Nachrichten nur für 30 Tage und danach nur noch die Anzahl.
[Benutzer können sich abmelden](#user-opt-out) und es werden niemals Inhalte von Nachrichten (oder Anhängen) gespeichert.
:::

## Statistiken aktivieren {#enable}

Um Statistiken zu aktivieren, besuche bitte die [Statistiken-Seite](https://scnx.app/de/glink?page=analytics) deines Servers auf SCNX und klicke auf
„Statistiken aktivieren“.

![](@site/docs/assets/scnx/guilds/analytics/setup.png)

## Verwendung im Dashboard {#dashboard}

Im [Statistiken-Dashboard](https://scnx.app/de/glink?page=analytics) kannst du die folgenden Daten basierend auf einem Zeitraum sehen, den Du oben in deinem Dashboard auswählen kannst:
* Nachrichten und Befehle: Du siehst ein Diagramm, das zeigt, wie viele Nachrichten und Befehle an jedem Tag im Zeitraum gesendet wurden.

> * 10 meist genutzte Kanäle: Dies ist eine Grafik, die zeigt, wie viel Prozent der Nachrichten über welchen Kanal gesendet wurden.

Reviewer 2: ```suggestion
* Zehn am meisten genutzte Kanäle: Dies ist eine Grafik, die zeigt, wie viel Prozent der Nachrichten in welchem Kanal gesendet wurden.
```

* Aktivste Benutzer: Das sind die 15. aktivsten Nutzer (die den [Datenschutzmodus](#user-opt-out) nicht aktiviert haben).
* Nutzerverlauf (immer letzte 30 Tage): Hier wird die Anzahl der Bots, Benutzer und aktive Mitglieder an jedem Tag des Zeitraums angezeigt.
* Aktivste Stunden: Diese Grafik zeigt, wie viele Nachrichten durchschnittlich zu welcher Zeit auf deinem Server gesendet wurden.
* Auf einen Blick: In diesem Abschnitt erhältst du einen schnellen Überblick über deinen Server. Folgende Daten werden angezeigt:
* Anzahl der im Zeitraum gesendeten Nachrichten
* Anzahl der im Zeitraum verwendeten Befehle
* Durchschnittliche Nachrichten pro Tag

> * Anzahl der aktiven Mitglieder auf deinen Server (ausgenommen Benutzer, die den [Datenschutzmodus] nicht aktiviert haben (#user-opt-out))

Reviewer: ```suggestion
* Anzahl der aktiven Mitglieder auf deinen Server (ausgenommen Benutzer, die den [Datenschutzmodus] aktiviert haben (#user-opt-out))
```

* Anzahl der Benutzer auf deinem Server
* Anzahl der Bots auf deinem Server

> *Bitte beachte, dass einige dieser Teile je nach Zeitrahmen möglicherweise nicht verfügbar sind.*

Reviewer: ```suggestion
*Bitte beachte, dass einige dieser Teile je nach Zeitraum möglicherweise nicht verfügbar sind.*
```

![](@site/docs/assets/scnx/guilds/analytics/dashboard.png)

## Analysen als Slash-Commands verwenden {#slash-commands}

Jeder Benutzer auf deinem Server kann die Slash-Befehle des verifizierten SCNX-Bots verwenden (du kannst dies unter
Discord-Server-Einstellungen unter „Integrationen“ konfigurieren). Folgende Befehle stehen zur Verfügung:

* `/stats guild`: Zeigt eine Statistikübersicht deines Servers an. Dazu gehören die 10 aktivsten Kanäle und Benutzer in den

> letzten 30 Tagen und einen Abschnitt mit einer „Zusammenfassung“, in der Sie die Gesamtzahl der Nachrichten und Befehle in den letzten 30 Tagen finden.

Reviewer: ```suggestion
letzten 30 Tagen und einen Abschnitt mit einer „Zusammenfassung“, in der du die Gesamtzahl der Nachrichten und Befehle in den letzten 30 Tagen findest.
```

> * `/stats user`: Zeigt eine Statistikübersicht eines Benutzers auf deinem Server an. Dazu gehören die 10 aktivsten Kanäle (und die Anzahl der darin gesendeten Nachrichten) sowie ein Abschnitt „Auf einen Blick“, indem du die Gesamtzahl der Nachrichten und Befehle der letzten 30 Tage findest.

Reviewer 2: ```suggestion
* `/stats user`: Zeigt eine Statistikübersicht eines Benutzers auf deinem Server an. Dazu gehören die zehn aktivsten Kanäle (und die Anzahl der darin gesendeten Nachrichten) sowie ein Abschnitt „Auf einen Blick“, indem du die Gesamtzahl der Nachrichten und Befehle der letzten 30 Tage findest.
```

> Benutzer, die den [Datenschutzmodus](#user-opt-out) aktiviert haben, können diesen Befehl nicht verwenden (/ zeigt Daten darüber an).

Reviewer 2: ```suggestion
Benutzer, die den [Datenschutzmodus](#user-opt-out) aktiviert haben, können diesen Befehl nicht verwenden.
```
oder besser anhand von original nochmal neu übersetzen

![](@site/docs/assets/scnx/guilds/analytics/commands.png)

## Fehlerbehebung {#troubleshooting}

<details>
    <summary>Nachrichten werden nicht im SCNX Analytics-Dashboard angezeigt</summary>
    <ul>

>          <li>Bitte stelle sicher, dass der (verifizierte) SCNX-Bot berechtigt ist, den Kanal anzuzeigen, in den die Nachricht gesendet wurden.

Reviewer: ```suggestion
         <li>Bitte stelle sicher, dass der (verifizierte) SCNX-Bot berechtigt ist, den Kanal zu sehen, in den die Nachricht gesendet wurden.
```

        </li>
        <li>Bitte beachte, dass das Dashboard ein Ergebnis einige Minuten lang zwischenspeichert. Bitte überprüfe den Zeitpunkt, an dem

>              das Ergebnis im Dashboard generiert wurde (dies wird oben auf der Seite angezeigt und sollte wie folgt aussehen: „Diese Zusammenfassung wurde
>             generiert um 11:30:28 Uhr").

Reviewer: ```suggestion
             das Ergebnis im Dashboard generiert wurde (dies wird oben auf der Seite angezeigt und sollte wie folgt aussehen: „Diese Zusammenfassung wurde frisch für dich
            um 11:30:28 Uhr generiert").
```

        </li>
          <li>Versuche, das Statistiken-Dashboard in ein paar Minuten zu aktualisieren.</li>
        <li>Benutzer, die den <a href="#user-opt-out">Datenschutzmodus</a> aktiviert haben, werden nicht im Dashboard angezeigt.
        </li>
    </ul>
</details>
<details>
    <summary>Ein Benutzer fehlt im Dashboard</summary>
    <ul>

>         <li>Stell sicher, dass der Benutzer eine Nachricht an einen Kanal gesendet hat, auf den der (verifizierte) SCNX-Bot zugriff hat.</li>

Reviewer: ```suggestion
        <li>Stelle sicher, dass der Benutzer eine Nachricht in einen Kanal gesendet hat, auf den der (verifizierte) SCNX-Bot Zugriff hat.</li>
```

                <li>Versuche, das Statistiken-Dashboard in ein paar Minuten zu aktualisieren.</li>
        <li>Benutzer, die den <a href="#user-opt-out">Datenschutzmodus</a> aktiviert haben, werden nicht im Dashboard angezeigt.
        </li>
    </ul>
</details>
<details>

>      <summary>Ich sehe die Meldung „Es gibt nicht viele Nachrichten im ausgewählten Zeitraum, daher sehen deine Diagramme möglicherweise etwas seltsam aus. ^^"  im Dashboard

Reviewer: ```suggestion
     <summary>Ich sehe die Meldung „Es existieren nur wenige Nachrichten im gewählten Zeitraum, deswegen können einige Diagramme merkwürdig aussehen ^^"  im Dashboard
```

         musst du Statistiken seit mindestens 48 Stunden aktiviert haben.“
    </summary>
    <ul>
        <li>Stelle sicher, dass Statistiken länger als 48 Stunden aktiviert sind.</li>
        <li>Aktualisiere das Dashboard.</li>
        <li>Stelle sicher, dass SCNX mindestens einen Kanal auf deinem Server sehen kann.</li>
    </ul>
</details>

## Benutzer-Opt-Out {#user-opt-out}

:::info Kurze Erinnerung
Wir speichern keine Nachrichteninhalte, sondern nur Metadaten zu Nachrichten (und das nur für 30 Tage nach dem Absenden einer Nachricht).
Alle infos findest du in unserem [FAQ-Artikel](https://faq.scnx.app/de/data-collection-with-analytics/).
:::

Wenn du (oder ein Benutzer deines Servers) nicht möchte, dass der Serverbesitzer und die Administratoren sehen können, wie viele Nachrichten du
gesendet hast oder du befürchtest, dass personenbezogene Daten über Dich oder deine Nutzer gespeichert werden, kannst du auf SCNX den „Datenschutzmodus“ aktivieren.
Verwende den Befehl `/privacy enable` auf deinem Server. Diese globale Aktion (das bedeutet, dass SCNX auf keinem Server Daten über dich speichert)
wird sich auf folgendes auswirken:

> * Alle Daten über von dir gesendete Nachrichten werden anonymisiert (= Deine Discord-Benutzer-ID wird aus der Datenbank entfernt)
>  - das bedeutet, dass wir nicht wissen, dass *du* eine Nachricht gesendet hast, sondern nur, dass eine Nachricht gesendet wurde.

Reviewer: ```suggestion
```

* Alle zukünftig von dir gesendeten Nachrichten werden anonymisiert gespeichert (= nicht mit deiner Discord-Benutzer-ID verknüpft) – dies
bedeutet, dass wir nicht wissen, dass *du* eine Nachricht gesendet hast, sondern nur, dass eine Nachricht gesendet wurde.
* Benutzer, die den Datenschutzmodus aktiviert haben, zählen weiterhin zu den Mitgliederaktivtitäts-Daten (da dies nicht die Speicherung von Discord-Benutzer-IDs beinhaltet)

> Der Datenschutzmodus kann mit (Opt-Back) mit „/privacy enable“ deaktiviert werden. Bitte beachte, dass dadurch frühere Nachrichten, die von dir gesendet wurden, nicht erneut verknüpft werden 

Reviewer: ```suggestion
Der Datenschutzmodus kann mit (Opt-Back) mit `/privacy disable` deaktiviert werden. Bitte beachte, dass dadurch frühere Nachrichten, die von dir gesendet wurden, nicht erneut verknüpft werden 
```

---
sidebar_position: 2
---

> # Serveranalyse

Reviewer: ```suggestion
# Serverstatistiken
```

> # Serveranalyse

Author: Fixed

Schalte detaillierte Statistiken über deinen Server frei – erhalte Infos, welche Mitglieder, Kanäle und an welchen Tagen auf deinem Server Aktivitäten sind.
Schau dir die Mitgliederaktivität an und erhalte eine Prognosen für die Zukunft 🔮

<IncludedInPlan data={{PRO: true, ACTIVE_GUILD: true, STARTER: true}}/>

> :::tip Der Datenschutz liegt uns am Herzen

Reviewer 3: ```suggestion
:::tip Datenschutz liegt uns am Herzen
```
In this case, this is a header of callout-box

> :::tip Der Datenschutz liegt uns am Herzen

Author: Fixed

> Du fragst dich, welche Daten wir über deinen Server speichern und wie wir sie verwenden? Bitte lese dir
> für weitere Informationen folgendes durch [FAQ-Artikel](https://faq.scnx.app/de/data-collection-with-analytics/).

Reviewer: ```suggestion
Du fragst dich, welche Daten wir über deinen Server speichern und wie wir sie verwenden? Bitte lese 
unseren [FAQ-Artikel](https://faq.scnx.app/datenerhebung-bei-statistiken/), um mehr zu erfahren.
```

> für weitere Informationen folgendes durch [FAQ-Artikel](https://faq.scnx.app/de/data-collection-with-analytics/).

Author: Fixed

> **tl;dr**: Wir speichern die Daten über Nachrichten nur 30 Tage lang und speichern die Zählungen erst danach
> [Benutzer können sich abmelden](#user-opt-out) und es werden niemals Inhalte von Nachrichten (oder Anhängen) gespeichert.

Reviewer: ```suggestion
**tl;dr**: Wir speichern Daten über Nachrichten nur für 30 Tage und dannach nur noch die Anzahl.
[Benutzer können sich abmelden](#user-opt-out) und es werden niemals Inhalte von Nachrichten (oder Anhängen) gespeichert.
```

> [Benutzer können sich abmelden](#user-opt-out) und es werden niemals Inhalte von Nachrichten (oder Anhängen) gespeichert.

Author: Fixed

:::

> ## Analyse aktivieren {#enable}

Reviewer: ```suggestion
## Statistiken aktivieren {#enable}
```

> ## Analyse aktivieren {#enable}

Author: Fixed

> Um Analysen zu aktivieren, besuche bitte die [Analyseseite](https://scnx.app/de/glink?page=analytics) Ihres Discords auf SCNX und klicke auf
> „Analyse aktivieren“.

Reviewer: ```suggestion
Um Statistiken zu aktivieren, besuche bitte die [Statistiken-Seite](https://scnx.app/de/glink?page=analytics) deines Servers auf SCNX und klicke auf
„Statistiken aktivieren“.
```

> „Analyse aktivieren“.

Author: Fixed

![](@site/docs/assets/scnx/guilds/analytics/setup.png)

## Verwendung im Dashboard {#dashboard}

> In deinem [Analyse-Dashboard](https://scnx.app/de/glink?page=analytics) kannst du die folgenden Daten basierend eines Zeitrahmens sehen, den Du oben in deinem Dashboard auswählen kannst:

Reviewer: ```suggestion
Im [Statistiken-Dashboard](https://scnx.app/de/glink?page=analytics) kannst du die folgenden Daten basierend auf einem Zeitraum sehen, den Du oben in deinem Dashboard auswählen kannst:
```

> In deinem [Analyse-Dashboard](https://scnx.app/de/glink?page=analytics) kannst du die folgenden Daten basierend eines Zeitrahmens sehen, den Du oben in deinem Dashboard auswählen kannst:

Author: Fixed

> * Nachrichten und Befehle: Du siehst ein Diagramm, das zeigt, wie viele Nachrichten und Befehle an jedem Tag im Zeitrahmen gesendet wurden.

Reviewer: ```suggestion
* Nachrichten und Befehle: Du siehst ein Diagramm, das zeigt, wie viele Nachrichten und Befehle an jedem Tag im Zeitraum gesendet wurden.
```

> * Nachrichten und Befehle: Du siehst ein Diagramm, das zeigt, wie viele Nachrichten und Befehle an jedem Tag im Zeitrahmen gesendet wurden.

Author: Fixed

> * 10 am häufigsten verwendete Kanäle: Dies ist eine Grafik, die zeigt, wie viel Prozent der Nachrichten über welchen Kanal gesendet wurden.

Reviewer: ```suggestion
* 10 meist genutzte Kanäle: Dies ist eine Grafik, die zeigt, wie viel Prozent der Nachrichten über welchen Kanal gesendet wurden.
```

> * 10 am häufigsten verwendete Kanäle: Dies ist eine Grafik, die zeigt, wie viel Prozent der Nachrichten über welchen Kanal gesendet wurden.

Author: Fixed

> * Aktivste Benutzer: Das sind die 15. aktivsten Nutzer (die den [Datenschutzmodus] (#user-opt-out) nicht aktiviert haben).

Reviewer: ```suggestion
* Aktivste Benutzer: Das sind die 15. aktivsten Nutzer (die den [Datenschutzmodus](#user-opt-out) nicht aktiviert haben).
```

> * Aktivste Benutzer: Das sind die 15. aktivsten Nutzer (die den [Datenschutzmodus] (#user-opt-out) nicht aktiviert haben).

Author: Fixed

> * Mitgliederaktivität (immer letzten 30 Tage): Hier wird die Anzahl der Bots, Benutzer und aktive Mitglieder an jedem Tag im Zeitrahmen angezeigt.

Reviewer: ```suggestion
* Nutzerverlauf (immer letzte 30 Tage): Hier wird die Anzahl der Bots, Benutzer und aktive Mitglieder an jedem Tag des Zeitraums angezeigt.
```

> * Mitgliederaktivität (immer letzten 30 Tage): Hier wird die Anzahl der Bots, Benutzer und aktive Mitglieder an jedem Tag im Zeitrahmen angezeigt.

Author: Fixed

> * Aktivste Stunden am Tag: Diese Grafik zeigt, wie viele Nachrichten durchschnittlich zu welcher Zeit auf Ihrem Server gesendet wurden.

Reviewer: ```suggestion
* Aktivste Stunden: Diese Grafik zeigt, wie viele Nachrichten durchschnittlich zu welcher Zeit auf deinem Server gesendet wurden.
```

> * Aktivste Stunden am Tag: Diese Grafik zeigt, wie viele Nachrichten durchschnittlich zu welcher Zeit auf Ihrem Server gesendet wurden.

Author: Fixed

> * Zusammenfassung: In diesem Abschnitt erhalten Sie einen schnellen Überblick über Ihren Server. Es umfasst folgende Daten:

Reviewer: ```suggestion
* Auf einen Blick: In diesem Abschnitt erhältst du einen schnellen Überblick über deinen Server. Folgende Daten werden angezeigt:
```

> * Zusammenfassung: In diesem Abschnitt erhalten Sie einen schnellen Überblick über Ihren Server. Es umfasst folgende Daten:

Author: Fixed

> * Anzahl der im Zeitrahmen gesendeten Nachrichten
> * Anzahl der im Zeitrahmen verwendeten Befehle
> * Durchschnittliche Nachrichten pro Tag
> * Anzahl der aktiven Mitglieder auf Ihrem Server (ausgenommen Benutzer, die den [Datenschutzmodus] nicht aktiviert haben (#user-opt-out))
> * Anzahl der Benutzer auf Ihrem Server
> * Anzahl der Bots auf Ihrem Server

Reviewer: ```suggestion
* Anzahl der im Zeitraum gesendeten Nachrichten
* Anzahl der im Zeitraum verwendeten Befehle
* Durchschnittliche Nachrichten pro Tag
* Anzahl der aktiven Mitglieder auf deinen Server (ausgenommen Benutzer, die den [Datenschutzmodus] nicht aktiviert haben (#user-opt-out))
* Anzahl der Benutzer auf deinem Server
* Anzahl der Bots auf deinem Server
```

> * Anzahl der Bots auf Ihrem Server

Author: Fixed

*Bitte beachte, dass einige dieser Teile je nach Zeitrahmen möglicherweise nicht verfügbar sind.*

![](@site/docs/assets/scnx/guilds/analytics/dashboard.png)

## Analysen als Slash-Commands verwenden {#slash-commands}

Jeder Benutzer auf deinem Server kann die Slash-Befehle des verifizierten SCNX-Bots verwenden (du kannst dies unter
Discord-Server-Einstellungen unter „Integrationen“ konfigurieren). Folgende Befehle stehen zur Verfügung:

* `/stats guild`: Zeigt eine Statistikübersicht deines Servers an. Dazu gehören die 10 aktivsten Kanäle und Benutzer in den
letzten 30 Tagen und einen Abschnitt mit einer „Zusammenfassung“, in der Sie die Gesamtzahl der Nachrichten und Befehle in den letzten 30 Tagen finden.

> * „/stats user“: Zeigt eine Statistikübersicht aller Benutzer auf deinem Server an. Dazu gehören die 10 aktivsten Kanäle (und die Anzahl der darin gesendeten Nachrichten) sowie ein Abschnitt „Zusammenfassung“, in dem Du die Gesamtzahl der Nachrichten und Befehle in den letzten 30 findest

Reviewer: ```suggestion
* `/stats user`: Zeigt eine Statistikübersicht eines Benutzer auf deinem Server an. Dazu gehören die 10 aktivsten Kanäle (und die Anzahl der darin gesendeten Nachrichten) sowie ein Abschnitt „Auf einen Blick“, indem du die Gesamtzahl der Nachrichten und Befehle der letzten 30 Tage findest.
```

> * „/stats user“: Zeigt eine Statistikübersicht aller Benutzer auf deinem Server an. Dazu gehören die 10 aktivsten Kanäle (und die Anzahl der darin gesendeten Nachrichten) sowie ein Abschnitt „Zusammenfassung“, in dem Du die Gesamtzahl der Nachrichten und Befehle in den letzten 30 findest

Author: Fixed

> Tage. Benutzer, die den [Datenschutzmodus] (#user-opt-out) aktiviert haben, können diesen Befehl nicht verwenden (/ zeigt Daten darüber an).

Reviewer: ```suggestion
Benutzer, die den [Datenschutzmodus](#user-opt-out) aktiviert haben, können diesen Befehl nicht verwenden (/ zeigt Daten darüber an).
```

> Tage. Benutzer, die den [Datenschutzmodus] (#user-opt-out) aktiviert haben, können diesen Befehl nicht verwenden (/ zeigt Daten darüber an).

Author: Fixed

![](@site/docs/assets/scnx/guilds/analytics/commands.png)

## Fehlerbehebung {#troubleshooting}

<details>
    <summary>Nachrichten werden nicht im SCNX Analytics-Dashboard angezeigt</summary>
    <ul>

>         <li>Bitte stell sicher, dass der (verifizierte) SCNX-Bot berechtigt ist, den Kanal anzuzeigen, über den die Nachricht gesendet wurden.

Reviewer: ```suggestion
        <li>Bitte stelle sicher, dass der (verifizierte) SCNX-Bot berechtigt ist, den Kanal anzuzeigen, in den die Nachricht gesendet wurden.
```

>         <li>Bitte stell sicher, dass der (verifizierte) SCNX-Bot berechtigt ist, den Kanal anzuzeigen, über den die Nachricht gesendet wurden.

Author: Fixed

        </li>
        <li>Bitte beachte, dass das Dashboard ein Ergebnis einige Minuten lang zwischenspeichert. Bitte überprüfe den Zeitpunkt, an dem

>             das Ergebnis im Dashboard generiert wurde ( dies wird oben auf der Seite angezeigt und sollte wie folgt aussehen: „Diese Zusammenfassung wurde

Reviewer: ```suggestion
            das Ergebnis im Dashboard generiert wurde (dies wird oben auf der Seite angezeigt und sollte wie folgt aussehen: „Diese Zusammenfassung wurde
```

>             das Ergebnis im Dashboard generiert wurde ( dies wird oben auf der Seite angezeigt und sollte wie folgt aussehen: „Diese Zusammenfassung wurde

Author: Fixed

            generiert um 11:30:28 Uhr").
        </li>

>         <li>Versuche, das Analyse-Dashboard in ein paar Minuten zu aktualisieren.</li>

Reviewer: ```suggestion
        <li>Versuche, das Statistiken-Dashboard in ein paar Minuten zu aktualisieren.</li>
```

>         <li>Versuche, das Analyse-Dashboard in ein paar Minuten zu aktualisieren.</li>

Author: Fixed

>         <li>Benutzer, für die der <a href="#user-opt-out">Datenschutzmodus</a> aktiviert ist, werden nicht im Dashboard angezeigt.

Reviewer: ```suggestion
        <li>Benutzer, die den <a href="#user-opt-out">Datenschutzmodus</a> aktiviert haben, werden nicht im Dashboard angezeigt.
```

>         <li>Benutzer, für die der <a href="#user-opt-out">Datenschutzmodus</a> aktiviert ist, werden nicht im Dashboard angezeigt.

Author: Fixed

        </li>
    </ul>
</details>
<details>
    <summary>Ein Benutzer fehlt im Dashboard</summary>
    <ul>
        <li>Stell sicher, dass der Benutzer eine Nachricht an einen Kanal gesendet hat, auf den der (verifizierte) SCNX-Bot zugriff hat.</li>

>         <li>Versuche, das Analyse-Dashboard in ein paar Minuten zu aktualisieren.</li>

Reviewer: ```suggestion
        <li>Versuche, das Statistiken-Dashboard in ein paar Minuten zu aktualisieren.</li>
```

>         <li>Versuche, das Analyse-Dashboard in ein paar Minuten zu aktualisieren.</li>

Author: Fixed

>         <li>Benutzer, für die der <a href="#user-opt-out">Datenschutzmodus</a> aktiviert ist, werden nicht im Dashboard angezeigt.

Reviewer: ```suggestion
        <li>Benutzer, die den <a href="#user-opt-out">Datenschutzmodus</a> aktiviert haben, werden nicht im Dashboard angezeigt.
```

>         <li>Benutzer, für die der <a href="#user-opt-out">Datenschutzmodus</a> aktiviert ist, werden nicht im Dashboard angezeigt.

Author: Fixed

        </li>
    </ul>
</details>
<details>

>     <summary>Ich sehe die Meldung „Es gibt nicht viele Nachrichten im ausgewählten Zeitraum, daher sehen deine Diagramme möglicherweise etwas seltsam aus.“
>         ^^" Warnung in deinem Dashboard

Reviewer: ```suggestion
    <summary>Ich sehe die Meldung „Es gibt nicht viele Nachrichten im ausgewählten Zeitraum, daher sehen deine Diagramme möglicherweise etwas seltsam aus. ^^"  im Dashboard
```

>         ^^" Warnung in deinem Dashboard

Author: Fixed

    </summary>
    <ul>

>         <li>Stell sicher, dass die Analyse länger als 48 Stunden aktiviert war.</li>
>         <li>Aktualisiere dein Dashboard.</li>
>         <li>Stell sicher, dass Du einen ausreichenden Zeitrahmen ausgewählt hast (z. B. „Letzte 30 Tage“ statt „Letzten 24 Stunden“).</li>

Reviewer: ```suggestion
        <li>Stelle sicher, dass Statistiken länger als 48 Stunden aktiviert sind.</li>
        <li>Aktualisiere dein Dashboard.</li>
        <li>Stelle sicher, dass Du einen ausreichenden Zeitraum ausgewählt hast (z. B. „Letzte 30 Tage“ statt „Letzte 24 Stunden“).</li>
```

>         <li>Stell sicher, dass Du einen ausreichenden Zeitrahmen ausgewählt hast (z. B. „Letzte 30 Tage“ statt „Letzten 24 Stunden“).</li>

Author: Fixed

>         <li>Stell sicher, dass im ausgewählten Zeitraum mehr als 100 Nachrichten gesendet wurden.</li>
>         <li>Bitte stell sicher, dass der (verifizierte) SCNX-Bot berechtigt ist, auf den Kanal zuzugreifen und den Nachrichtenverlauf sehen kann, über den die Nachrichten gesendet wurden.

Reviewer: ```suggestion
        <li>Stelle sicher, dass im ausgewählten Zeitraum mehr als 100 Nachrichten gesendet wurden.</li>
        <li>Bitte stelle sicher, dass der (verifizierte) SCNX-Bot auf den Kanal zugreifen kann und den Nachrichtenverlauf sehen kann, über den die Nachrichten gesendet wurden.
```

>         <li>Bitte stell sicher, dass der (verifizierte) SCNX-Bot berechtigt ist, auf den Kanal zuzugreifen und den Nachrichtenverlauf sehen kann, über den die Nachrichten gesendet wurden.

Author: Fixed

        </li>
    </ul>
</details>
<details>

>     <summary>Anstatt deine Mitgliederaktivität zu sehen, zeigt dir das Dashboard diese Warnung an: „Um Daten über deine Mitglieder anzuzeigen,
>         musst du die Analyse seit mindestens 48 Stunden aktiviert haben.“
>     </summary>
>     <ul>
>         <li>Stell sicher, dass die Analyse länger als 48 Stunden aktiviert war.</li>
>         <li>Aktualisieren das Dashboard.</li>
>         <li>Stell sicher, dass SCNX mindestens einen Kanal auf deinem Server sehen und zugreifen kann.</li>

Reviewer: ```suggestion
    <summary>Anstatt deine Mitgliederaktivität zu sehen, zeigt dir das Dashboard diese Warnung an: „Um Daten über deine Mitglieder anzuzeigen,
        musst du Statistiken seit mindestens 48 Stunden aktiviert haben.“
    </summary>
    <ul>
        <li>Stelle sicher, dass Statistiken länger als 48 Stunden aktiviert sind.</li>
        <li>Aktualisiere das Dashboard.</li>
        <li>Stelle sicher, dass SCNX mindestens einen Kanal auf deinem Server sehen kann.</li>
```

>         <li>Stell sicher, dass SCNX mindestens einen Kanal auf deinem Server sehen und zugreifen kann.</li>

Author: Fixed

    </ul>
</details>

## Benutzer-Opt-Out {#user-opt-out}

:::info Kurze Erinnerung
Wir speichern keine Nachrichteninhalte, sondern nur Metadaten zu Nachrichten (und das nur für 30 Tage nach dem Absenden einer Nachricht).

> Alle infos findest Du unter [FAQ-Artikel](https://faq.scnx.app/de/data-collection-with-analytics/).

Reviewer: ```suggestion
Alle infos findest du in unserem [FAQ-Artikel](https://faq.scnx.app/de/data-collection-with-analytics/).
```

> Alle infos findest Du unter [FAQ-Artikel](https://faq.scnx.app/de/data-collection-with-analytics/).

Author: Fixed

:::

> Wenn Du (oder ein Benutzer deines Servers) nicht möchte, dass der Serverbesitzer und die Administratoren sehen können, wie viele Nachrichten Du erhalten oder
> gesendet hast oder Du befürchtest, dass personenbezogene Daten über Dich oder deine Nuztzer gespeichert werden, kannst du auf SCNX den „Datenschutzmodus“ aktivieren.

Reviewer: ```suggestion
Wenn du (oder ein Benutzer deines Servers) nicht möchte, dass der Serverbesitzer und die Administratoren sehen können, wie viele Nachrichten du
gesendet hast oder du befürchtest, dass personenbezogene Daten über Dich oder deine Nutzer gespeichert werden, kannst du auf SCNX den „Datenschutzmodus“ aktivieren.
```

> gesendet hast oder Du befürchtest, dass personenbezogene Daten über Dich oder deine Nuztzer gespeichert werden, kannst du auf SCNX den „Datenschutzmodus“ aktivieren.

Author: Fixed

> Verwenden folgenden Befehl „/privacy enable“ auf deinem Server. Das ist global (das bedeutet, dass SCNX auf keinem Server Daten über Dich speichert)
> wird sich auf Folgendes auswirken:

Reviewer: ```suggestion
Verwende den Befehl `/privacy enable` auf deinem Server. Diese globale Aktion (das bedeutet, dass SCNX auf keinem Server Daten über dich speichert)
wird sich auf folgendes auswirken:
```

> wird sich auf Folgendes auswirken:

Author: Fixed

> * Alle Daten über von Dir gesendete Nachrichten werden anonymisiert (= Deine Discord-Benutzer-ID wird aus der Datenbank entfernt)

Reviewer: ```suggestion
* Alle Daten über von dir gesendete Nachrichten werden anonymisiert (= Deine Discord-Benutzer-ID wird aus der Datenbank entfernt)
```

> * Alle Daten über von Dir gesendete Nachrichten werden anonymisiert (= Deine Discord-Benutzer-ID wird aus der Datenbank entfernt)

Author: Fixed

> - das bedeutet, dass wir nicht wissen, dass *Du* eine Nachricht gesendet hast, sondern nur, dass eine Nachricht gesendet wurde.

Reviewer: ```suggestion
 - das bedeutet, dass wir nicht wissen, dass *du* eine Nachricht gesendet hast, sondern nur, dass eine Nachricht gesendet wurde.
```

> - das bedeutet, dass wir nicht wissen, dass *Du* eine Nachricht gesendet hast, sondern nur, dass eine Nachricht gesendet wurde.

Author: Fixed

> * Alle zukünftig von Dir gesendeten Nachrichten werden anonymisiert gespeichert (= nicht mit Deiner Discord-Benutzer-ID verknüpft) – dies
> bedeutet, dass wir nicht wissen, dass *Du* eine Nachricht gesendet hast, sondern nur, dass eine Nachricht gesendet wurde.

Reviewer: ```suggestion
* Alle zukünftig von dir gesendeten Nachrichten werden anonymisiert gespeichert (= nicht mit deiner Discord-Benutzer-ID verknüpft) – dies
bedeutet, dass wir nicht wissen, dass *du* eine Nachricht gesendet hast, sondern nur, dass eine Nachricht gesendet wurde.
```

> bedeutet, dass wir nicht wissen, dass *Du* eine Nachricht gesendet hast, sondern nur, dass eine Nachricht gesendet wurde.

Author: Fixed

* Benutzer, die den Datenschutzmodus aktiviert haben, zählen weiterhin zu den Mitgliederaktivtitäts-Daten (da dies nicht die Speicherung von Discord-Benutzer-IDs beinhaltet)

> Datenschutzmodus kann mit (Opt-Back) mit „/privacy enable“ deaktiviert werden. Bitte beachte, dass dadurch Frühere Nachrichten nicht erneut verknüpft werden
> Die von dir gesendete wurden (das ist nicht mehr möglich), d. h., dass nur neue Nachrichten in Befehlen und im Dashboard angezeigt werden.

Reviewer: ```suggestion
Der Datenschutzmodus kann mit (Opt-Back) mit „/privacy enable“ deaktiviert werden. Bitte beachte, dass dadurch frühere Nachrichten, die von dir gesendet wurden, nicht erneut verknüpft werden 
(das ist nicht mehr möglich), d. h., dass nur neue Nachrichten in Befehlen und im Dashboard angezeigt werden.
```

> Die von dir gesendete wurden (das ist nicht mehr möglich), d. h., dass nur neue Nachrichten in Befehlen und im Dashboard angezeigt werden.

Author: Fixed
